src/styles/untils/blocks/stopwatch.scss

```diff
@@ -0,0 +1,39 @@
+.stopwatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  &__second {
+    position: absolute;
+    top: $mainTop;
+    width: 8px;
+    height: $mainHeight;
```

Comment:
these values (in addition to `width`), are used 2 times (in `&__second` and `&__minute`), so they can be exported to the mixin